tools/install-powershell.readme.md

```diff
@@ -0,0 +1,56 @@
+
+## Features of install-powershell.sh
```

Comment:
@TravisEz13 - I'm not sure what you mean.  "##" is level two heading.

I also don't know what you mean by "Can you please enable these files in this test..."  - how would I do this?

Please point me to a markdown validator where I can reproduce the syntax errors or the failed test report.

Thanks,
D.